jbpm-wb-common/jbpm-wb-common-client/src/test/java/org/jbpm/workbench/common/client/menu/ManageSelectorImplTest.java

```diff
@@ -0,0 +1,135 @@
+ * limitations under the License.
+ */
+
+package org.jbpm.workbench.common.client.menu;
+
+import com.google.gwt.dev.util.collect.Sets;
+import com.google.gwtmockito.GwtMockitoTestRunner;
+import org.jboss.errai.security.shared.api.identity.User;
+import org.jbpm.workbench.common.client.PerspectiveIds;
+import org.jbpm.workbench.common.client.resources.i18n.Constants;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.uberfire.client.mvp.Activity;
+import org.uberfire.client.mvp.ActivityManager;
+import org.uberfire.client.mvp.PerspectiveActivity;
+import org.uberfire.client.mvp.PerspectiveManager;
+import org.uberfire.client.mvp.PlaceManager;
+import org.uberfire.mvp.Command;
+import org.uberfire.mvp.impl.DefaultPlaceRequest;
+import org.uberfire.security.ResourceRef;
+import org.uberfire.security.authz.AuthorizationManager;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.*;
+
+@RunWith(GwtMockitoTestRunner.class)
+public class ManageSelectorImplTest {
+
+    private String selected_Perspective = PerspectiveIds.PROCESS_DEFINITIONS;
+
+    @InjectMocks
+    ManageSelectorImpl manageSelector;
+
+    @Mock
+    ManageSelectorImpl.ManageSelectorView view;
+
+    @Mock
+    PerspectiveManager perspectiveManagerMock;
+
+    @Mock
+    PerspectiveActivity perspectiveActivityMock;
+
+    @Mock
+    private PlaceManager placeManagerMock;
+
+    @Mock
+    private ActivityManager activityManagerMock;
+
+    @Mock
+    private AuthorizationManager authorizationManager;
+
+    @Mock
+    private User identity;
+
+    @Before
+    public void setup() {
+        when(perspectiveActivityMock.getIdentifier()).thenReturn(selected_Perspective);
+        when(perspectiveManagerMock.getCurrentPerspective()).thenReturn(perspectiveActivityMock);
+        when(view.getSelectedOption()).thenReturn(selected_Perspective);
+        when(activityManagerMock.getActivities(any(DefaultPlaceRequest.class))).thenReturn(Sets.create(mock(Activity.class)));
+    }
+
+    @Test
+    public void testInitManageSelector_AllPerspectivesGranted() {
+        when(authorizationManager.authorize(any(ResourceRef.class),
+                                            eq(identity))).thenReturn(true);
+        manageSelector.init();
+        verifyInitSelectorBasicViewInteractions();
+        verify(view).addOption(Constants.INSTANCE.Process_Definitions(),
+                               PerspectiveIds.PROCESS_DEFINITIONS,
+                               true);
+        verify(view).addOption(Constants.INSTANCE.Process_Instances(),
+                               PerspectiveIds.PROCESS_INSTANCES,
+                               false);
+        verify(view).addOption(Constants.INSTANCE.Tasks(),
+                               PerspectiveIds.TASKS_ADMIN,
```

Comment:
@cristianonicolai @nmirasch One unrelated question. 
Do we want to continue having the perspective Id named TASKS_ADMIN?

Reply:
@bsremac I think we could rename that on master to the names are closer to the screen name.